### PR TITLE
Feature/#2019 token rule widget

### PR DESCRIPTION
**Pull request by aminbksi**

### CODE REVIEW

File: src/core/interfaces/tokenRules.interface.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {TokenRuleListHeaderType} from 'core/types';
+
+export interface ColumnHeaderInterface {
```
Comment (Daboo1998) [inline src/core/interfaces/tokenRules.interface.ts:3]: Name of the file doesn't match the interface name at all, please use consistent naming

File: src/core/types/tokenRules.type.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export type TokenRuleListHeaderType = 'icon' | 'name' | 'status' | 'info';
```
Comment (Daboo1998) [inline src/core/types/tokenRules.type.ts:1]: Again constant naming

File: src/scenes/spaceAdmin/pages/SpaceAdminPage/components/templates/index.ts [hunk 3]
```diff
@@ -2,6 +2,6 @@ export * from './SpaceDetailsPanel';
 export * from './SpaceMembersPanel';
 export * from './SubSpacesPanel';
 export * from './TokenRulesPanel';
-export * from './TokenRuleForm';
-export * from './TokenForm';
-export * from './ApplyTokenRuleForm';
+export * from './TokenRulesPanel/components/templates/TokenRuleForm';
+export * from './TokenRulesPanel/components/templates/TokenForm';
+export * from './TokenRulesPanel/components/templates/ApplyTokenRuleForm';
```
Comment (Daboo1998) [inline src/scenes/spaceAdmin/pages/SpaceAdminPage/components/templates/index.ts:7]: No, remove those then, and export in `./TokenRulesPanel/components/templates/index.ts` file

File: src/scenes/widgets/Widgets.tsx [hunk 4]
```diff
@@ -72,9 +69,9 @@ const Widgets: FC = () => {
     userDevicesStore.toggleCamera();
   };
 
-  const handleRuleReviewClose = () => {
-    tokenRulesStore.tokenRuleReviewDialog.close();
-    tokenRulesStore.tokenRulesListStore.fetchTokenRules();
+  const handleOpenProfileMenu = () => {
```
Comment (PAKovalev) [inline src/scenes/widgets/Widgets.tsx:72]: Move this function to store

File: src/scenes/widgets/Widgets.tsx [hunk 5]
```diff
@@ -168,7 +158,7 @@ const Widgets: FC = () => {
           {/* Main toolbar icons */}
           <ToolbarIconList>
             {currentProfile?.profile && (
-              <ToolbarIcon title="Profile" onClick={profileMenuDialog.open}>
+              <ToolbarIcon title="Profile" onClick={handleOpenProfileMenu}>
```
Comment (PAKovalev) [inline src/scenes/widgets/Widgets.tsx:161]: i18n

**Review by Daboo1998 — CHANGES_REQUESTED** [review]

**Review by PAKovalev — APPROVED** [review]

**Review by Daboo1998 — APPROVED** [review]

**Pull request merged**